content/docs/actors/development/secret_input.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Secret input
+description: Learn about making some input fields secret and encrypted.
+paths:
+# NOTE: IF ADDING A NEW PATH, LEAVE THE OLD ONES FOR REDIRECTS
+    - actor/secret-input
+    - actors/secret-input
+    - actor/development/secret-input
+    - actors/development/secret-input
+---
+
+# [](#secret-input)Secret input
+
+The Secret Input feature allows you to make an input field secret, causing it to be stored encrypted when setting the actor input, and decrypted only inside the actor.
+
+## Setting an input field as secret
+
+To make an input field secret, just set `"isSecret": true` to the field in the actor's [input schema]({{@link actors/development/input_schema.md}}), like this:
+
+<!-- eslint-skip -->
+```json
+{
+    // ...
+    "properties": {
+        // ...
+        "password": {
+            "title": "Password",
+            "type": "string",
+            "description": "A secret, encrypted input field",
+            "editor": "textfield",
+            "isSecret": true
+        },
+        // ...
+    },
+    // ...
+}
+```
+
+The editor for this input field will then turn into a secret input, and when you edit the field value, it will be stored encrypted.
+<img src="{{@asset actors/development/images/secret-input-editor.webp}}" alt="Secret input editor" style="width: 100%; max-width: 822px;"/>
+
+This is only available for `string` inputs, and the editor type is limited to `textfield` or `textarea`.
+
+## Reading secret input fields
+
+When you read the actor input through `Actor.getInput()`, the encrypted fields are automatically decrypted, without any additional code needed (starting with the [`apify` package](https://www.npmjs.com/package/apify) version 3.1.0).
```

Comment:
Is this also available in Crawlee? `apify` is the Apify SDK, right?

Reply:
It's not in Crawlee, because that gets the actor input through the Apify SDK. I didn't want to put Apify SDK here, since in the Apify SDK for Python, this will be available right from the start (and it will probably start at version 1, and have different versioning than the JS SDK).